kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/HostapdConfigWriter.java

```diff
@@ -316,11 +316,11 @@ private String updateInterfaceAndDriver(String interfaceName, String interfaceDr
     }
 
     private String updateChannels(WifiConfig wifiConfig, String fileAsString) throws KuraException {
-        if (wifiConfig.getChannels()[0] > 0 && wifiConfig.getChannels()[0] < 14) {
+        if (wifiConfig.getChannels()[0] > 0 && wifiConfig.getChannels()[0] <= 14) {
```

Comment:
I think you forgot to change the header in this file, it should be "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"